app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1158,6 +1178,10 @@ class BrowserTabFragment :
                 includeShortcutToViewCredential = it.includeShortcutToViewCredential,
             )
 
+            is Command.WebViewError -> {
+                showError(it.errorType)
+            }
+
```

Comment:
```suggestion
            is Command.WebViewError -> showError(it.errorType)
```
NIT: Format consistency with existing `when` cases